### PR TITLE
Disallow administration of users having permissions that the logged-i…

### PR DESCRIPTION
**Pull request by sunsetsystems**
…n user does not have, and assignment of such permissions.

#### Short description of what this resolves:
This opens the possibility to give admin/users permission to non-superusers, allowing them to create and administer users who do not exceed their own permissions.

### CODE REVIEW

File: interface/usergroup/usergroup_admin.php [hunk 1]
```diff
@@ -184,9 +186,12 @@
                     );
                 }
                 $olduf["$facid/$whid"] = false;
+                /******************************************************
+                // What's the intent here? $deffacid does not appear anywhere else in the code.
                 if ($facid == $deffacid) {
                     $deffacid = 0;
                 }
+                ******************************************************/
```
Comment (bradymiller) [inline interface/usergroup/usergroup_admin.php:194]: vote to remove it :)

dwight-schrute-the-office

**Comment by sjpadgett** [comment]
pretty cool.

**Comment by sunsetsystems** [comment]
Thanks Jerry.
I'll bring this in later today if there are no objections.

**Comment by sjpadgett** [comment]
None by me Rod.

**Pull request merged**